src/shared/ui/Dialog/Dialog.jsx

```diff
@@ -0,0 +1,19 @@
+import { Portal } from "react-portal";
+import "./Dialog.css";
+
+export const Dialog = ({ title, children, onCloseCB }) => {
```

Comment:
Why isn't just `onClose` ?
```suggestion
export const Dialog = ({ title, children, onClose }) => {
```

Reply:
sorry, it is an old code, done